utils/umaApi.ts

```diff
@@ -221,6 +222,11 @@ export async function fetchCompleteSynth<T extends { type: ContractType }>(
   }
 }
 
-export function formatLSPName(longTokenName: string): string {
-  return longTokenName.substring(-2);
+// longTokenName can return undefined if value wasn't found in API so we need to do a null check.
```

Comment:
imho it would be better to do this check at the top level and keep this function only accepting strings, we can default it at the top level to an empty string or a placeholder